### PR TITLE
Update PWA-AEP integration

### PR DESCRIPTION
**Pull request by erikmarr**
## Purpose of this pull request

This pull request (PR) updates the PWA-AEP integration topic to reference the new feature name: Data Connection.

## Affected pages

- https://developer.adobe.com/commerce/pwa-studio/integrations/adobe-commerce/aep/

### CODE REVIEW

**Comment by erikmarr** [comment]
Converting to draft as this should not be merged until Tuesday, November 14th.

**Review by jhadobe — APPROVED** [review]

**Review by hguthrie — APPROVED** [review]
Looks good. 
Will those links work Nov 14?

**Comment by erikmarr** [comment]
Thanks for the review @hguthrie! Yes, those links will be live on the 14th.

**Comment by keharper** [comment]
@erikmarr Are there any upgrade instructions for those already using the Experience Platform Connector?

File: src/pages/integrations/adobe-commerce/aep/index.md [hunk 1]
```diff
@@ -9,37 +9,37 @@ keywords:
 
 # Adobe Experience Platform
 
-Included in the PWA Studio 12.5.0 release is an eventing framework that lets store owners install extensions that track user interaction within their site and collect analytics information.
-With the Experience Platform Connector extension, Adobe Commerce users with an Adobe Experience Platform (AEP) subscription can get event data from PWA Studio storefronts and process it on the AEP.
+Included in the PWA Studio 12.5.0 release is an eventing framework that lets merchants install extensions that track user interaction within their site and collect analytics information.
+With the Data Connection extension, Adobe Commerce users with an Adobe Experience Platform (AEP) subscription can get event data from PWA Studio storefronts and process it on the Experience Platform.
```
Comment (erikmarr) [inline src/pages/integrations/adobe-commerce/aep/index.md:13]:
```suggestion
With the [!DNL Data Connection] extension, Adobe Commerce users with an Adobe Experience Platform (AEP) subscription can get event data from PWA Studio storefronts and process it on the Experience Platform.
```

File: src/pages/integrations/adobe-commerce/aep/index.md [hunk 2]
```diff
@@ -9,37 +9,37 @@ keywords:
...
 
 This integration requires the following pieces to function properly:
 
-- The Experience Platform Connector extension for PWA Studio storefronts, which connects and send data to the AEP platform.
-- The Experience Platform extensions installed in the Adobe Commerce backend which provides configuration information to the storefront extension.
+- The Data Connection extension for PWA Studio storefronts, which connects and sends data to the Experience Platform.
```
Comment (erikmarr) [inline src/pages/integrations/adobe-commerce/aep/index.md:17]:
```suggestion
- The [!DNL Data Connection] extension for PWA Studio storefronts, which connects and sends data to the Experience Platform.
```

File: src/pages/integrations/adobe-commerce/aep/index.md [hunk 3]
```diff
@@ -9,37 +9,37 @@ keywords:
...
 ![Basic event flow](images/analytics-flow.png)
 
 The following sections provide installation and configuration instructions for the Experience Platform integration.
-Magento Open Source does not support the AEP.
+Magento Open Source does not support the Experience Platform.
 
 ## Install the Experience Platform extension for PWA Studio
 
-Use your storefront project's package manager(`node` or `yarn`), to install the Experience Platform Connector extension into your storefront project.
+Use your storefront project's package manager(`node` or `yarn`), to install the Data Connection extension into your storefront project.
```
Comment (erikmarr) [inline src/pages/integrations/adobe-commerce/aep/index.md:29]:
```suggestion
Use your storefront project's package manager(`node` or `yarn`), to install the [!DNL Data Connection] extension into your storefront project.
```

File: src/pages/integrations/adobe-commerce/aep/index.md [hunk 4]
```diff
@@ -9,37 +9,37 @@ keywords:
...
 
 ```terminal
-yarn add @magento/experience-platform-connector
+yarn add @magento/data-connection
 ```
 
 ## Adobe Commerce backend configuration
 
-There are few steps to take to establish a data stream and then install and configure the Experience Platform Connector.
+There are few steps to take to establish a data stream and then install and configure the Data Connection extension.
```
Comment (erikmarr) [inline src/pages/integrations/adobe-commerce/aep/index.md:39]:
```suggestion
There are few steps to take to establish a data stream and then install and configure the [!DNL Data Connection]  extension.
```

File: src/pages/integrations/adobe-commerce/aep/index.md [hunk 5]
```diff
@@ -9,37 +9,37 @@ keywords:
...
 
-Check the prerequisites for establishing a data stream in the [Experience Platform Connector overview](https://experienceleague.adobe.com/docs/commerce-merchant-services/experience-platform-connector/overview.html?lang=en#prereqs).
+Check the prerequisites for establishing a data stream in the [Data Connection overview](https://experienceleague.adobe.com/docs/commerce-merchant-services/data-connection/overview.html#prereqs).
```
Comment (erikmarr) [inline src/pages/integrations/adobe-commerce/aep/index.md:41]:
```suggestion
Check the prerequisites for establishing a data stream in the [[!DNL Data Connection] overview](https://experienceleague.adobe.com/docs/commerce-merchant-services/data-connection/overview.html#prereqs).
```

File: src/pages/integrations/adobe-commerce/aep/index.md [hunk 6]
```diff
@@ -9,37 +9,37 @@ keywords:
...
 
-To install the EPC within Adobe Commerce, see [Install and Configure the Experience Platform Connector](https://experienceleague.adobe.com/docs/commerce-merchant-services/experience-platform-connector/fundamentals/install.html?lang=en).
+To install the EPC within Adobe Commerce, see [Install and Configure the Data Connection extension](https://experienceleague.adobe.com/docs/commerce-merchant-services/data-connection/fundamentals/install.html).
```
Comment (erikmarr) [inline src/pages/integrations/adobe-commerce/aep/index.md:43]:
```suggestion
To install the EPC within Adobe Commerce, see [Install and Configure the [!DNL Data Connection] extension](https://experienceleague.adobe.com/docs/commerce-merchant-services/data-connection/fundamentals/install.html).
```

**Review by erikmarr — COMMENTED** [review]
Adding DNL tags around feature name.

**Comment by hguthrie** [comment]
> Adding DNL tags around feature name.

@erikmarr  DNL tags are for Experience League. I don't think they will provide the same effect in the devsite.

**Comment by erikmarr** [comment]
@keharper there are no specific upgrade steps for existing users. Since this is a SaaS delivered feature, the Admin will display the new name. The package names are not changing.

**Comment by erikmarr** [comment]
@hguthrie I'll remove the DNL tags. Thanks!

File: src/pages/integrations/adobe-commerce/aep/index.md [hunk 7]
```diff
@@ -9,37 +9,37 @@ keywords:
 
 # Adobe Experience Platform
 
-Included in the PWA Studio 12.5.0 release is an eventing framework that lets store owners install extensions that track user interaction within their site and collect analytics information.
-With the Experience Platform Connector extension, Adobe Commerce users with an Adobe Experience Platform (AEP) subscription can get event data from PWA Studio storefronts and process it on the AEP.
+Included in the PWA Studio 12.5.0 release is an eventing framework that lets merchants install extensions that track user interaction within their site and collect analytics information.
+With the [!DNL Data Connection] extension, Adobe Commerce users with an Adobe Experience Platform (AEP) subscription can get event data from PWA Studio storefronts and process it on the Experience Platform.
```
Comment (erikmarr) [inline src/pages/integrations/adobe-commerce/aep/index.md:13]:
```suggestion
With the Data Connection extension, Adobe Commerce users with an Adobe Experience Platform (AEP) subscription can get event data from PWA Studio storefronts and process it on the Experience Platform.
```

File: src/pages/integrations/adobe-commerce/aep/index.md [hunk 8]
```diff
@@ -9,37 +9,37 @@ keywords:
...
 
 This integration requires the following pieces to function properly:
 
-- The Experience Platform Connector extension for PWA Studio storefronts, which connects and send data to the AEP platform.
-- The Experience Platform extensions installed in the Adobe Commerce backend which provides configuration information to the storefront extension.
+- The [!DNL Data Connection] extension for PWA Studio storefronts, which connects and sends data to the Experience Platform.
```
Comment (erikmarr) [inline src/pages/integrations/adobe-commerce/aep/index.md:17]:
```suggestion
- The Data Connection extension for PWA Studio storefronts, which connects and sends data to the Experience Platform.
```

File: src/pages/integrations/adobe-commerce/aep/index.md [hunk 9]
```diff
@@ -9,37 +9,37 @@ keywords:
...
 
 ```terminal
-yarn add @magento/experience-platform-connector
+yarn add @magento/data-connection
 ```
 
 ## Adobe Commerce backend configuration
 
-There are few steps to take to establish a data stream and then install and configure the Experience Platform Connector.
+There are few steps to take to establish a data stream and then install and configure the [!DNL Data Connection]  extension.
```
Comment (erikmarr) [inline src/pages/integrations/adobe-commerce/aep/index.md:39]:
```suggestion
There are few steps to take to establish a data stream and then install and configure the Data Connection extension.
```

File: src/pages/integrations/adobe-commerce/aep/index.md [hunk 10]
```diff
@@ -9,37 +9,37 @@ keywords:
...
 
-Check the prerequisites for establishing a data stream in the [Experience Platform Connector overview](https://experienceleague.adobe.com/docs/commerce-merchant-services/experience-platform-connector/overview.html?lang=en#prereqs).
+Check the prerequisites for establishing a data stream in the [[!DNL Data Connection] overview](https://experienceleague.adobe.com/docs/commerce-merchant-services/data-connection/overview.html#prereqs).
 
-To install the EPC within Adobe Commerce, see [Install and Configure the Experience Platform Connector](https://experienceleague.adobe.com/docs/commerce-merchant-services/experience-platform-connector/fundamentals/install.html?lang=en).
+To install the EPC within Adobe Commerce, see [Install and Configure the [!DNL Data Connection] extension](https://experienceleague.adobe.com/docs/commerce-merchant-services/data-connection/fundamentals/install.html).
```
Comment (erikmarr) [inline src/pages/integrations/adobe-commerce/aep/index.md:43]:
```suggestion
To install the EPC within Adobe Commerce, see [Install and Configure the Data Connection extension](https://experienceleague.adobe.com/docs/commerce-merchant-services/data-connection/fundamentals/install.html).
```

**Review by erikmarr — COMMENTED** [review]
removing DNL tags.

**Pull request merged**